apps/src/lib/kits/maker/dropletConfig.js

```diff
@@ -263,7 +285,15 @@ const circuitPlaygroundBlocks = [
   ...sharedLedBlocks({
     category: CIRCUIT_CATEGORY,
     blockPrefix: colorLedBlockPrefix,
-    objectDropdown: {options: colorPixelVariables}
+    objectDropdown: {options: colorPixelVariables},
+    includePrefixInFunc: true
+  }),
+
+  ...sharedLedBlocks({
```

Comment:
Perhaps we could label these sharedColorLedBlocks since there are functions with the same name for color LEDS vs LEDs.

Your explanation of how the block color helps me understand why the .on block is blue in the Maker drawer when Micro:Bit is enabled since there is no colorLED .on block for the Micro:Bit. Thanks for all the context!

![Screen Shot 2023-02-13 at 5 22 17 PM](https://user-images.githubusercontent.com/107423305/218597722-d1c9a559-0325-4b9f-881e-83dd4868d28b.png)

Reply:
Sure, renamed!